client/alloc_runner.go

```diff
@@ -344,7 +343,6 @@ OUTER:
 		case update := <-r.updateCh:
 			// Check if we're in a terminal status
 			if update.TerminalStatus() {
-				r.setAlloc(update)
```

Comment:
Why did we remove this? I think no one else is even calling it or referencing it, probably we should delete the method if this isn't needed anymore?